inst/templates/app_structure/dot.lintr

```diff
@@ -1,5 +1,5 @@
 linters:
-  with_defaults(
+  linters_with_defaults(
     line_length_linter = line_length_linter(100),
     infix_spaces_linter = NULL,
     object_usage_linter = NULL
```

Comment:
1. In version 3.0.0 an exception was added for `/` usage in `box::use()` so we can now reenable the `infix_sapces_linter`. I learned this from the [changelog](https://lintr.r-lib.org/news/index.html).
2. Let's add a comment why `object_usage_linter` is disabled.
```suggestion
    object_usage_linter = NULL  # Does not work with `box::use()`.
```